http/version/parse_test.go

```diff
@@ -0,0 +1,14 @@
+package version
+
+import (
+	"testing"
+)
+
+func TestParseHttpVersion(t *testing.T) {
```

Comment:
Can refer to https://github.com/FlowerLab/sorbifolia/blob/67cca0dc31d6e414735b104456254bbc9b67d1a1/http/httpmessage/request_write_test.go